src/Storages/StorageTinyLog.cpp

```diff
@@ -294,11 +294,15 @@ IDataType::OutputStreamGetter TinyLogBlockOutputStream::createStreamGetter(
 void TinyLogBlockOutputStream::writeData(const String & name, const IDataType & type, const IColumn & column, WrittenStreams & written_streams)
 {
     IDataType::SerializeBinaryBulkSettings settings;
-    settings.getter = createStreamGetter(name, written_streams);
 
     if (serialize_states.count(name) == 0)
+    {
+        WrittenStreams prefix_written_streams;
```

Comment:
Could you please add a comment - many people will read this code as an example...

Reply:
Added